inst/sql/sqlite/migrations/Migration_2-v3_1_0_ir_person_years.sql

```diff
@@ -0,0 +1,26 @@
+-- changes incidence_rate.person_years from bigint to float
+{DEFAULT @migration = migration}
+{DEFAULT @incidence_rate = incidence_rate}
+{DEFAULT @table_prefix = ''}
+
+-- ALTER TABLE @results_schema.@table_prefix@incidence_rate ALTER COLUMN person_years FLOAT;
+
+ALTER TABLE @database_schema.@table_prefix@incidence_rate RENAME TO _incidence_rate_old;
+
+CREATE TABLE @database_schema.@table_prefix@incidence_rate (
+    cohort_count BIGINT NOT NULL,
+    person_years NUMERIC,
+    gender VARCHAR,
+    age_group VARCHAR,
+    calendar_year VARCHAR(4),
```

Comment:
Same comment about an unbounded VARCHAR, and should calendar year be varchar?  Not only is an int smaller to store, but it would also prevent the storage of things like 'NA' or 'xxxx'

Reply:
I'm not sure about the reason these are VARCHAR - it dates back to a design decision made before I was working on this (he're I literally copy pasted the table definition for sqlite). I assume this is because of how the data was stored. I'd suggest moving away in a future version, for now I have just bounded them to values.